crates/bevy_ui/src/ui_transform.rs

```diff
@@ -189,3 +227,30 @@ impl From<&UiGlobalTransform> for Affine2 {
         value.0
     }
 }
+
+impl Mul<UiGlobalTransform> for UiGlobalTransform {
+    type Output = Affine2;
```

Comment:
Just curious, why is the Output not another `UiGlobalTransform`?

Same with the Affine * UiGlobalTransform.

Reply:
It just seems more conveniant, there aren't really any occasions where you'd want a `UiGlobalTransform` back from one of these operations. I think maybe it's more natural if `UiGlobalTransform` * `UiGlobalTransform` returns a `UiGlobalTransform`.